Sample.java

```diff
@@ -1,7 +1,88 @@
-// Time Complexity :
-// Space Complexity :
-// Did this code successfully run on Leetcode :
-// Any problem you faced while coding this :
+// Time Complexity : (n)^2
+// Space Complexity : n
+// Did this code successfully run on Leetcode : no
+// Any problem you faced while coding this : solution looks working theoretically have to learn to code and debug without debugger
 
 
 // Your code here along with comments explaining your approach
+Problem1
+
+class MyQueue {
+
+    /** Initialize your data structure here. */
+    public Stack<Integer> stack1,stack2;
+    public MyQueue() {
+        stack1 = new Stack<>();
+        stack2 = new Stack<>();
+    }
+
+    /** Push element x to the back of queue. */
+    public void push(int x) {
+        if(empty()){
+            stack1.push(x);
+        }else if(stack1.size() == 0 && stack2.size() != 0) {
+            loan();
+            stack1.push(x);
+        }else {
+            stack1.push(x);
+        }
+    }
+
+    public void loan(){
+        if(stack1.size() == 0 && stack2.size() !=0) {
+            for(int i = 0 ; i < stack2.size() ; i++ ){
```

Comment:
Consider the size of stack 2 is 6
if you use for loop, you only pop out 3 elements. 
Every time you pop, stack2 size decreases and i increases. That's the reason why you won't be able to get the minimum elements.